pro_1.py

```diff
@@ -0,0 +1,100 @@
+import json
+import Patterns
+import config_st
+import database_interaction
+import User
+import re 
+from re import findall
+database_file=open("database.txt","r+")
+dict_n=json.load(database_file)
+
+
+def logged_in():
+    print("succes")
+
+def log_in():
+    print("login page")
+def sign_up():
+   
+    print("=====signup page=====")
+
+    def enter_ur_name():
+     name=input("step1/4----enter ur full name ")
+     i=0
+     while(len(name)==0 or name==" ") :
```

Comment:
this can be broken by two spaces!
try to look up a string method called strip

Reply:
yah i use it thx